web_src/css/dashboard.css

```diff
@@ -92,6 +92,6 @@
 
 @media (max-width: 767.98px) {
   .dashboard .dashboard-navbar .ui.dropdown > .menu {
-    position: static;
+    position: absolute;
```

Comment:
IIRC the default is "absolute" for menu item.

Reply:
shell i remove this css rule?

Reply:
If it duplicates, why not?

I guess there is some reason for using "static" here, maybe you can try to blame the history.

Reply:
i will share my findings... on this

Reply:
Here is the origin of position 

https://github.com/puni9869/gitea/commit/202803fc69d21763a06d8d0f5a4c46509c18f6f1#diff-809c14c56f75fdeee75e624f39fbfa7225c8f8225962aa7e8213944153a75cdaR102

Its not duplicate and was `static` from starting.

Reply:
I believe it's not the root position.

Reply:
The root is here: Fix dashboard switching on Mobile (#20238)

![image](https://github.com/go-gitea/gitea/assets/2114189/4e09d722-aa9d-4c3e-b25f-cbb553aaddb9)

Reply:
Thanks for pointing me to the room.

A lot has been change after that in popups and dropdowns. We had a huge fix in mobile view in issue filter dropdowns. We had same problem where we fix the position of element to `absolute`.

At present the current UI is stable in mobile. I did a regression check in all the screens last night.

Reply:
Could we just remove this "absolute" style if the default style is already "absolute"?

Reply:
Working, nicly without absolute too.

Reply:
![image](https://github.com/go-gitea/gitea/assets/80308335/be0edcb5-c0d0-4a36-aae3-ed178789f683)